babel/dates.py

```diff
@@ -1336,7 +1350,7 @@ def get_week_number(self, day_of_period, day_of_week=None):
     'h': [1, 2], 'H': [1, 2], 'K': [1, 2], 'k': [1, 2],             # hour
     'm': [1, 2],                                                    # minute
     's': [1, 2], 'S': None, 'A': None,                              # second
-    'z': [1, 2, 3, 4], 'Z': [1, 2, 3, 4], 'v': [1, 4], 'V': [1, 4]  # zone
+    'z': [1, 2, 3, 4, 5], 'Z': [1, 2, 3, 4, 5], 'v': [1, 4], 'V': [1, 4],  'x': [1, 2, 3, 4, 5], 'X': [1, 2, 3, 4, 5]# zone
```

Comment:
The new characters (`x`, `X`) should also be included in `PATTERN_CHAR_ORDER`, below.